src/classes/ADDR_Contact_TDTM.cls

```diff
@@ -50,168 +51,195 @@ public class ADDR_Contact_TDTM extends TDTM_Runnable {
     * @return dmlWrapper.  
     ********************************************************************************************************/
     public override DmlWrapper run(List<SObject> listNew, List<SObject> listOld, 
-        TDTM_Runnable.Action triggerAction, Schema.DescribeSObjectResult objResult) {
+    TDTM_Runnable.Action triggerAction, Schema.DescribeSObjectResult objResult) {
             
-        DmlWrapper dmlWrapper = new DmlWrapper();        
-        list<Contact> listConAddrSet = new list<Contact>(); 
-        list<Contact> listConCreateAddr = new list<Contact>();
-        map<Id, boolean> mapAddrIdIsOverride = new map<Id, boolean>();
-        list<Contact> listConAddrReset = new list<Contact>();
-        Map<Id,Account> mapAccountIdAccount = null;
-        
-        // bail out if address mgmt turned off.
-        if (String.isBlank(UTIL_CustomSettingsFacade.getSettings().Accounts_Addresses_Enabled__c))
-            return dmlWrapper;
-                
-        // Rules:
-        // inserting new contact - make their address a new default address, unless they say it is an override
-        // updating an existing contact - make their address a new default address, unless they say it is an override
-        
-        boolean isBefore = (triggerAction == TDTM_Runnable.Action.BeforeUpdate || triggerAction == TDTM_Runnable.Action.BeforeInsert);
-        if (isBefore) { 
-            // need to query account fields we need to look at
-            Set<Id> setAccountId = new Set<Id>();
-            for (SObject so : listNew) {
-                Contact con = (Contact)so;
-                if (con.AccountId != null)
-                    setAccountId.add(con.AccountId);
-            }
-            mapAccountIdAccount = new Map<Id,Account>([select Id, RecordTypeId from Account where Id IN :setAccountId]);
-        }        
+        DmlWrapper dmlWrapper = new DmlWrapper();  
+        List<Contact> listcontactCreateAddr = new List<Contact>();      
+        List<Contact> listContactsAddrInfoCleared = new List<Contact>();
        
-        integer i = -1;        
```

Comment:
This variable doesn't seem to have a very meaningful name. I suggest to give this variable a concise but meaningful name that suggest its purpose.